packages/di/inversify/package.json

```diff
@@ -36,16 +36,15 @@
   "dependencies": {
     "@suites/types.common": "^3.0.0",
     "@suites/types.di": "^3.0.0",
-    "lodash.isequal": "^4.5.0"
+    "lodash.isequal": "^4.5.0",
+    "reflect-metadata": "^0.1.13"
```

Comment:
Why in prod dependencies? We want the user to be able to choose the version no?

Reply:
The library consumes reflect-metadata so it must be installed, if it will be a devDep it won't be installed for the user. Suites "production" is a devDependency for another project.